.restyled.yaml

```diff
@@ -1,21 +1,14 @@
 restylers:
   - name: prettier
-    arguments:
-      [
-        '--write',
-        '--arrow-parens=avoid',
-        '--print-width=80',
-        '--prose-wrap=always',
-        '--no-semi',
-        '--single-quote=true',
-        '--tab-width=2',
-        '--trailing-comma=none',
-        '--use-tabs=false'
-      ]
-    include: # matching '**/*.{js,jsx,md,tsx,ts,json}'
-      - '**/*.md'
-      - '**/*.jsx'
-      - '**/*.tsx'
+    arguments: ['--write']
+    include:
+      - '**/*.css'
       - '**/*.js'
-      - '**/*.ts'
       - '**/*.json'
+      - '**/*.jsx'
+      - '**/*.md'
+      - '**/*.mdx'
+      - '**/*.ts'
+      - '**/*.tsx'
+      - '**/*.yaml'
+      - '**/*.yml'
```

Comment:
Updated restyled config to not have its own unique options as well as include all applicable files. I _think_ this brings it in line with local `prettier`, but I can't find anything truly confirming this in writing, but Restyled not finding any issues despite our non-default pretter config seems like evidence this is the case.